binary-search.py

```diff
@@ -834,9 +834,17 @@ def run_trial (trial_params, port_info, stream_info, detailed_stats):
         flow_mods_opt = ' --flowMods="' + re.sub('^,', '', flow_mods_opt) + '"'
         cmd = cmd + flow_mods_opt
     elif trial_params['traffic_generator'] == 'xena':
-         # TODO: pass binary search trial parameters to external helper module (xena-txrx.py)
-         cmd = 'python -u ' + t_global.trafficgen_dir + 'xena-txrx.py'
-         sys.exit()
+        # required trial arguments
+        cmd = 'python -u ' + t_global.trafficgen_dir + 'xena-txrx.py'
+        cmd = cmd + ' --active-device-pairs=' + str(trial_params['active_device_pairs'])
+        cmd = cmd + ' --chasis=' + str(trial_params['chasis'])
```

Comment:
should be 'chassis'

Reply:
Requested changes implemented. Thanks for the typo catch!